sdk/python/feast/registry.py

```diff
@@ -523,10 +558,12 @@ def update_registry_proto(
             registry_proto.registry_schema_version = REGISTRY_SCHEMA_VERSION
         if updater:
             registry_proto = updater(registry_proto)
-        self._write_registry(registry_proto)
+        self._write_registry_blob(registry_proto)
+        if self.publish_json:
```

Comment:
There seems to be an inconsistency here. In the one case `_write_registry_json` is inside `_write_registry_blob` and in the other case outside.

Reply:
oh wow, thanks for catching this - totally meant to put it in `update_registry_proto` for both cases. Inconsistency bugs me a ton too so not sure how I managed to do this.

I'll be putting both inside `update_registry_proto`